content/millennium/dstu2/medications/medication-statement.md

```diff
@@ -88,17 +89,20 @@ _Implementation Notes_
 ### Parameters
 
 
- Name            | Required?          | Type          | Description
------------------|--------------------|---------------|--------------------------------------------------------------------------------------------------------
- `_id`           | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
- `patient`       | This, or `_id`     | [`reference`] | The identifier of a patient to list statements for. Example: `12345`
- `status`        | N                  | [`token`]     | The status of the medication statement, may be a list separated by commas. Example: `active,completed`
- [`_count`]      | N                  | [`number`]    | The maximum number of results to return. Defaults to `50`.
+ Name        | Required?     | Type          | Description
+------------ |---------------|---------------|--------------------------------------------------------------------------------------------------------
+ `_id`       | Conditionally | [`token`]     | The logical resource ID associated with the resource. It may be a list separated by commas. This parameter is required if the `patient` parameter is not used. Example: `_id=1234`
+ `patient`   | Conditionally | [`reference`] | The specific patient to return medication statements for. This parameter is required if the `_id` parameter is not used. Example: `patient=5678`
+ `status`    | No            | [`token`]     | The status of the medication statement, may be a list separated by commas. Example: `status=active,completed`
+ [`_count`]  | No            | [`number`]    | The maximum number of results to return. Defaults to `_count=50`.
 
 
-Notes:
+_Implementation Notes_
```

Comment:
The recommendation between Notes/Implementation Notes from tech writers is to use Notes